User event handler v2

loofen opened the pull request:
- Fixed form. All except address which can be handled later adding maps etc.
- Added New event button in nav-bar
- Edit / Destroy buttons for user who own event
- New/create and edit/update working
- Added time to date

loofen committed:
added form for new and edit event. edit/destroy button on show page

loofen committed:
Fixed points from last review, except Address field which we can fix later

nathansoussana commented:
Looks good @loofen! Tested the functionality and it works great.

I would approve and merge now, but I would still consider small improvements on the front-end side:

- When selecting "Open to all" levels, The view says "forall", not formatted properly for a view. We might want to change that
- The width of the form on wider screens: It appears quite wide on my Mac, we might want to make it not as wide on wider screens, but full width on mobile. You might want to look at Bootstrap for that.
- In order to make the form not too plain, we could consider a background image in the same way as on some other pages.

Most of these are just suggestions, feel free to push back if you disagree.

Thank you!

loofen committed:
fixed format on form, added banners to edit and new.

loofen commented:
Fixed the format on new and edit.

Added banners to new and edit.

I fint front-end-wise a background behind the forms might get a bit much. But maybe we can update the banner pictures later.

nathansoussana committed:
Added padding to the left and the right side on the form for mobile and centered title in the new and edit banners

nathansoussana reviewed (approved):
Nice edits! Just added some padding to the form to optimise the mobile view. Thank you!
